.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+## Contributing
+
+First, thanks for considering contributing to our project! In general, our
+`Development.md` contains a lot of development-related information that could
+prove useful to a contributor!
+
+### Issues
+
+Before creating a new issue, look at current and past issues both on Github
```

Comment:
add links to: GitHub issues, JIRA project, outstanding PRs

Reply:
Done.